modules/appeals_api/config/schemas/v2/10182.json

```diff
@@ -0,0 +1,150 @@
+                },
+                "extensionRequest": {
+                  "type": "boolean",
+                  "description": "Request for an extension of time to file"
+                 },
+                "extensionReason": {
+                  "type": "string",
+                  "maxLength": 1500,
+                  "description": "Explanation why you believe there is good cause for the extension."
+                },
+                "appealsingVhaDenial": {
```

Comment:
Though I love `appealsing` and believe it should be added to the English language dictionary, I think it should be corrected!

Reply:
Thank you for catching this!